generate_workspace/src/main/java/com/google/devtools/build/workspace/maven/Resolver.java

```diff
@@ -321,6 +322,14 @@ private String downloadSha1(Rule rule) {
     return null;
   }
 
+  /**
+   * Prints currently downloaded maven artifact to console.
+   */
+  private static void logArtifactDownload(Model model) {
+    System.out.println("\tDownloading maven artifact " + model.getGroupId() + ":"
```

Comment:
I think the time is actually useful.  Don't worry about the verbosity for now, at least it's tunable this way.